rules/game_process.tex

```diff
@@ -75,6 +78,8 @@ \subsection{Robot States}
   \label{fig:robot_states}
 \end{figure}
 
+The referee must tell the GameController operator to switch to \texttt{initial} state once everyone is in place or that teams are falling under forfeit rule (see \cref{sec:forfeit}).
```

Comment:
Should we explain here that both halves start with the setup state? 

Previous sections already imply it, though the text is kinda vague. In the past the rules also didn't write more explicitly that halves start with initial. 

opinions?